plugins/no_sir.py

```diff
@@ -0,0 +1,15 @@
+from errbot import BotPlugin
+
+
+class no_sir(BotPlugin):
+    """do not use sir to sound more friendly"""  # Ignore QuotesBear
+
+    def callback_message(self, msg):
+        emots = [':D']
+        if match_sir:
+            self.send(
+                msg.frm,
+                '@{}, donot use \'sir\' in ur conversation. {}'.format(
```

Comment:
This indentation looks quite weird and it's hard to read it, try to make it a bit more compact :)

Reply:
k